bridges/snowbridge/pallets/outbound-queue-v2/src/envelope.rs

```diff
@@ -16,15 +15,15 @@ sol! {
 
 /// An inbound message that has had its outer envelope decoded.
 #[derive(Clone, RuntimeDebug)]
-pub struct Envelope<T: Config> {
+pub struct Envelope {
 	/// The address of the outbound queue on Ethereum that emitted this message as an event log
 	pub gateway: H160,
 	/// A nonce for enforcing replay protection and ordering.
 	pub nonce: u64,
 	/// Delivery status
 	pub success: bool,
 	/// The reward address
-	pub reward_address: T::AccountId,
+	pub reward_address: [u8; 32],
```

Comment:
Curious why this change?

Reply:
It's for fixing some compile issue as mentioned in https://github.com/vgeddes/polkadot-sdk/pull/1#discussion_r1939576476

Since the reward address is on AH, which can only be a 32-byte address, this should be fine. 

Or we may need to introduce an explicit converter from `T::AccountId` to `Location` same as https://github.com/paritytech/polkadot-sdk/pull/7402#discussion_r1941218780